modules/hal_silabs/wiseconnect/CMakeLists.txt

```diff
@@ -142,3 +142,6 @@ zephyr_library_sources_ifdef(CONFIG_WISECONNECT_NETWORK_STACK
   ${PROTOCOL_DIR}/wifi/si91x/sl_wifi.c
   ${PROTOCOL_DIR}/wifi/src/sl_wifi_callback_framework.c
 )
+
+zephyr_linker_sources(ROM_SECTIONS linker/code_classification_text.ld)
+zephyr_linker_sources(RAMFUNC_SECTION linker/code_classification_ramfunc.ld)
```

Comment:
This code is duplicated with simplicity_sdk. I have to work on that.

(In fact, I get errors from `sl_core_cortexm.c`, but I have not yet understand how this file come into the compilation.)